diff --git a/dist/index.d.ts b/dist/index.d.ts
index f4adce20734c5bd272bcbb810fd200d8c91fd335..8382ad6c2b01c16efa2bc71f632e4f6867636fcb 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -43,5 +43,4 @@ declare function viteLegacyPlugin(options?: Options): Plugin[];
 declare function detectPolyfills(code: string, targets: any, assumptions: Record<string, boolean>, list: Set<string>): Promise<void>;
 declare const cspHashes: string[];
 
-export = viteLegacyPlugin;
-export { type Options, cspHashes, detectPolyfills };
+export { type Options, cspHashes, detectPolyfills, viteLegacyPlugin as default };
